### PR TITLE
Cron v10

### PR DESCRIPTION
**Pull request by creativcoder**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: cron_v10/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+authors = [
+  "ChainSafe Systems <info@chainsafe.io>",
+  "Protocol Labs",
+  "Filecoin Core Devs",
+]
+edition = "2018"
+repository = "https://github.com/filecoin-project/builtin-actors"
+keywords = ["filecoin", "web3", "wasm"]
+
+[lib]
```
Comment (tyshko5) [inline cron_v10/Cargo.toml:15]: I think we don't need this

File: cron_v10/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+use fvm_shared::METHOD_CONSTRUCTOR;
+use num_derive::FromPrimitive;
+use num_traits::Zero;
+
+pub use self::state::{Entry, State};
+
+mod state;
+pub mod testing;
+
+#[cfg(feature = "fil-actor")]
```
Comment (tyshko5) [inline cron_v10/src/lib.rs:19]: We don't have this feature

File: cron_v10/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#[derive(Default, Debug, Serialize_tuple, Deserialize_tuple)]
+pub struct ConstructorParams {
+    /// Entries is a set of actors (and corresponding methods) to call during EpochTick.
+    pub entries: Vec<Entry>,
+}
+
+/// Cron actor
+pub struct Actor;
+
+impl Actor {
```
Comment (tyshko5) [inline cron_v10/src/lib.rs:43]: We don't need to have anything connected to the `Actor`

**Review by tyshko5 — CHANGES_REQUESTED** [review]

File: cron_v10/Cargo.toml (hunk 1 above)
Comment (creativcoder) [reply cron_v10/Cargo.toml]: removed

File: cron_v10/src/lib.rs (hunk 2 above)
Comment (creativcoder) [reply cron_v10/src/lib.rs]: removed

File: cron_v10/src/lib.rs (hunk 3 above)
Comment (creativcoder) [reply cron_v10/src/lib.rs]: I see, removed

File: cron_v10/src/lib.rs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+/// Constructor parameters for Cron actor, contains entries
+/// of actors and methods to call on each epoch
+#[derive(Default, Debug, Serialize_tuple, Deserialize_tuple)]
+pub struct ConstructorParams {
+    /// Entries is a set of actors (and corresponding methods) to call during EpochTick.
+    pub entries: Vec<Entry>,
+}
+
+/// Cron actor
+pub struct Actor;
```
Comment (tyshko5) [inline cron_v10/src/lib.rs:32]: We don't need this as well

**Review by tyshko5 — CHANGES_REQUESTED** [review]

File: cron_v10/src/lib.rs (hunk 3 above)
Comment (tyshko5) [reply cron_v10/src/lib.rs]: Not completely

File: cron_v10/src/lib.rs (hunk 4 above)
Comment (creativcoder) [reply cron_v10/src/lib.rs]: removed

File: cron_v10/src/lib.rs (hunk 3 above)
Comment (creativcoder) [reply cron_v10/src/lib.rs]: updated with changes

**Pull request merged**